src/multiint.cpp

```diff
@@ -4742,9 +4302,33 @@ void WzMultiplayerOptionsTitleUI::addPlayerBox(bool players)
 	W_LABEL* pPlayersLabel = addSideText(FRONTEND_SIDETEXT2, MULTIOP_PLAYERSX - 3, MULTIOP_PLAYERSY, _("PLAYERS"));
 	pPlayersLabel->hide(); // hide for now
 
-	if (players)
+	if (!players)
+	{
+		return;
+	}
+
+	auto titleUI = std::dynamic_pointer_cast<WzMultiplayerOptionsTitleUI>(shared_from_this());
```

Comment:
Should be sufficient and safe enough to invoke `static_pointer_cast` here, since we know for sure that `this` represents a `WzMultiplayerOptionsTitleUI` instance.